src/rgw/rgw_cache.h

```diff
@@ -157,15 +157,15 @@ struct ObjectCacheEntry {
 class ObjectCache {
   std::unordered_map<string, ObjectCacheEntry> cache_map;
```

Comment:
@cbodley Let's inherit from config observer api here. We could convert rgw_cache_lru_size and rgw_cache_expiry interval.